config/app/README.md

```diff
@@ -8,4 +8,13 @@ This is the entry point to the main server setup.
 
 ## Init
 Contains a list of initializer that need to be run when starting the server.
-* *default*: The default setup that makes sure the root container has the necessary resources.
+* *default*: The default setup that makes sure the server starts up correctly.
+             The ParallelHandler can be used to add custom Initializers.
```

Comment:
Can we be more precise: what does "start up correctly" mean?

i.e., at the moment, this specifier doesn't add meaning, because nobody would expect the default configuration to ensure that the server starts incorrectly 😉

Alternatively, this phrase can just become

```suggestion
* *default*: The default setup; ParallelHandler can be used to add custom Initializers.
```